nupic/research/frameworks/pytorch/imagenet/parser_utils.py

```diff
@@ -0,0 +1,165 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import argparse
+import socket
+
+import torch
+from wandb import util
+
+from nupic.research.frameworks.dynamic_sparse.common.ray_custom_loggers_2 import (
+    DEFAULT_LOGGERS,
+)
+from nupic.research.frameworks.pytorch.imagenet import mixins
+from nupic.research.frameworks.sigopt.sigopt_experiment import SigOptImagenetExperiment
+from nupic.research.frameworks.wandb import ray_wandb
+
```

Comment:
Could you add `__all__` list here exporting the parsers and the helper functions?